src/contribute/release-guide.md

```diff
@@ -511,7 +512,220 @@ If there are no issues, reply on the vote thread to close the voting. Then, tall
+            -DarchetypeCatalog=internal
+            
+      mvn compile exec:java -Dexec.mainClass=org.apache.beam.examples.complete.game.injector.Injector \
+        -Dexec.args="${YOUR_PROJECT} ${YOUR_PUBSUB_TOPIC} none"
+      ```
+  * Run Leaderboard with Direct Runner
+    ```
+    python -m apache_beam.examples.complete.game.leader_board \
+    --project=${YOUR_PROJECT} \
+    --topic projects/${YOUR_PROJECT}/topics/${YOUR_PUBSUB_TOPIC} \
+    --dataset ${USER}_test
+    ```
+    Inspect results:
+    * Check whether there is any error messages in console.
+    * Goto your BigQuery console and check whether your ${USER}_test has leader_board_users and leader_board_teams table.
+    * bq head -n 10 ${USER}_test.leader_board_users
+    * bq head -n 10 ${USER}_test.leader_board_teams
+  * Run Leaderboard with Dataflow Runner
+    ```
+    python -m apache_beam.examples.complete.game.leader_board \ 
+    --project=${YOUR_PROJECT} \ 
+    --topic projects/${YOUR_PROJECT}/topics/${YOUR_PUBSUB_TOPIC} \ 
+    --dataset ${USER}_test \ 
+    --runner DataflowRunner \ 
+    --temp_location=${YOUR_GS_BUCKET}/temp/ \ 
+    --sdk_location dist/*
+    ```
+    Inspect results:
+    * Goto your Dataflow job console and check whether there is any error.
+    * Goto your BigQuery console and check whether your ${USER}_test has leader_board_users and leader_board_teams table.
+    * bq head -n 10 ${USER}_test.leader_board_users
+    * bq head -n 10 ${USER}_test.leader_board_teams
+  * Run GameStats with Direct Runner
+    ```
+    python -m apache_beam.examples.complete.game.game_stats \
+    --project=${YOUR_PROJECT} \
+    --topic projects/${YOUR_PROJECT}/topics/${YOUR_PUBSUB_TOPIC} \
+    --dataset ${USER}_test \
+    --fixed_window_duration ${SOME_SMALL_DURATION}
+    ```
+    Inspect results:
+    * Check whether there is any error messages in console.
+    * Goto your BigQuery console and check whether your ${USER}_test has game_stats_teams and game_stats_sessions table.
+    * bq head -n 10 ${USER}_test.game_stats_teams
+    * bq head -n 10 ${USER}_test.game_stats_sessions
+
+  * Run GameStats with Dataflow Runner
+    ```
+    python -m apache_beam.examples.complete.game.game_stats \ 
+    --project=${YOUR_PROJECT} \ 
+    --topic projects/${YOUR_PROJECT}/topics/${YOUR_PUBSUB_TOPIC} \ 
+    --dataset ${USER}_test \ 
+    --runner DataflowRunner \ 
+    --temp_location=${YOUR_GS_BUCKET}/temp/ \ 
+    --sdk_location dist/* \
+    --fixed_window_duration ${SOME_SMALL_DURATION}
+    ```
+    Inspect results:
+    * Goto your Dataflow job console and check whether there is any error.
+    * Goto your BigQuery console and check whether your ${USER}_test has game_stats_teams and game_stats_sessions table.
+    * bq head -n 10 ${USER}_test.game_stats_teams
+    * bq head -n 10 ${USER}_test.game_stats_sessions
 ### Checklist to proceed to the finalization step
 
 1. Community votes to release the proposed candidate, with at least three approving PMC votes
```

Comment:
All community members are encouraged to vote, with +1 to approve the release and -1 if there is an issue with the release. If there is an issue, clarify in the email. Only PMC members can cast binding votes.